tests/BookingInfoTest.php

```diff
@@ -6,6 +6,7 @@
 
 use CultuurNet\UDB3\Model\ValueObject\Contact\BookingAvailability;
 use CultuurNet\UDB3\Model\ValueObject\Contact\TelephoneNumber;
+use CultuurNet\UDB3\Model\ValueObject\Translation\Language as Udb3ModelLanguage;
```

Comment:
I'm not totally sure why you do this rename

Reply:
This can indeed be removed, must be something temporary I forgot to remove again.